ontology/yaml/resources/LIGHTING/entity_types/ABSTRACT.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+### ABSTRACT TYPES
+
+SS:
+  description: "Basic combination of run command and status (start/stop); indicates the light is active or inactive."
+  is_abstract: true
+  uses:
+  - run_command
+  - run_status
+
+DC:
+  description: "Brightness dimming control."
+  is_abstract: true
+  uses:
+  - brightness_percentage_command
```

Comment:
Technically yes, but the reason for doing that in the HVAC namespace was to reduce complexity... this namespace has substantially less data than HVAC, and so if we were sticklers on that point, we'd end up with effectively no abstract concepts (since there is little in common between the fields here) and would be defining them on canonical concepts. My guess is also that there will be more added to these abstract types once we get more variety from different systems we integrate.